### ISSUE TITLE
Add landmark roles and aria-labels to nav elements

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/rockarch.org#49]
For screen readers, each `<nav>` element should have a landmark role and an `aria-label` that describes the nav:

- top <nav> needs `role="navigation"` and `aria-label="main"`.
- breadcrumb needs `role="navigation"` (it already had an aria-label)
- side nav should have its role changed to "navigation" and add `aria-label="page content"`
